src/file_manager.cpp

```diff
@@ -2,54 +2,58 @@
 
 namespace big
 {
-    bool file_manager::init(const std::filesystem::path& base_dir)
-    {
-        m_base_dir = base_dir;
-        file_manager::ensure_folder_exists(m_base_dir);
-
-        return true;
-    }
-
-    const std::filesystem::path& file_manager::get_base_dir()
-    {
-        return m_base_dir;
-    }
-    
+	bool file_manager::init(const std::filesystem::path& base_dir)
+	{
+		m_base_dir       = base_dir;
```

Comment:
Revert these changes, you are not supposed to touch this class.

Reply:
Yep leave this class alone, see a later comment on how to properly make use of the file_manager.